### PR TITLE
GitHub Actions to update errata.tex

### PR DESCRIPTION
**Pull request by favonia**
While we are deciding the fate of nightly builds in #1103, I believe we can already enable the updater for `errata.tex`. This script will be triggered by (1) manual execution, (2) any commit to the master branch, and (3) any pull request from the same repo (for example this one). The commit message will be `Mark Errata (auto)` as the one used by the original Travis script.

The code was extracted from #1102. I tried to minimize changes to other files in this PR.

~~PS: Currently, I [manually disabled this action](https://github.com/HoTT/book/actions/workflows/errata.yml) because otherwise it would _immediately_ try to update `errata.tex` in this PR.~~

### CODE REVIEW

**Comment by mikeshulman** [comment]
Seems ok as far as I can tell.  Do we just have to merge it and see whether it works?

**Comment by JasonGross** [comment]
> PS: Currently, I [manually disabled this action](https://github.com/HoTT/book/actions/workflows/errata.yml) because otherwise it would _immediately_ try to update `errata.tex` in this PR.

What's wrong with it updating the errata in this PR?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -56,9 +46,3 @@ jobs:
           
     - name: Build
       run: make ${{ matrix.targets }}
-    
-    - name: Update errata
-      run: etc/ci/update_errata.sh
- 
-    - name: Update nightlies
```
Comment (JasonGross) [inline .github/workflows/ci.yml:63]: For this PR, we should leave in the updating of nightlies, right?

**Comment by favonia** [comment]
> > PS: Currently, I [manually disabled this action](https://github.com/HoTT/book/actions/workflows/errata.yml) because otherwise it would _immediately_ try to update `errata.tex` in this PR.
> 
> What's wrong with it updating the errata in this PR?

I prefer updating it in the master branch instead of this PR because I didn't want to make the impression that this PR has directly changed errata.tex in any way. That said, this is a very minor point that I don't feel strongly about. There's nothing really "wrong".

File: .github/workflows/ci.yml (hunk 1 above)
Comment (favonia) [reply .github/workflows/ci.yml:63]: No, I think we should pause the updating until we have decided what to do and then completed the transition. The current code is already not working (well), and the redirect server is down. The updater had stopped working for more than one year and I believe we can wait a bit longer. Running the semi-working updater for nightly builds _now_ is not very useful IMO.

**Comment by favonia** [comment]
> Seems ok as far as I can tell. Do we just have to merge it and see whether it works?

Yes. In fact, it already worked on this PR, but then I manually reverted the automatic commit.

**Comment by favonia** [comment]
@JasonGross @mikeshulman Alright, I re-enabled the workflow so that, as you can see, the script managed to update `errata.tex` in this PR as 574a6d8251c4a6a29f687707d8958b85080eaee7. It is easy to change the detail of the automatic commit---including setting its author to the one who originally triggers the event. I still feel it is better to put a hold on updating nightly builds until #1103 is resolved.

**Review by mikeshulman — APPROVED** [review]

**Pull request merged**